pkg/utils/flow/flow.go

```diff
@@ -101,7 +99,7 @@ func (n *node) addTargets(taskIDs ...TaskID) {
 // are left blank and don't affect the Flow.
 type Opts struct {
 	Logger           logrus.FieldLogger
-	ProgressReporter func(ctx context.Context, stats *Stats)
+	ProgressReporter ProgressReporter
```

Comment:
This is a breaking change for extensions using the flow pkg. Can you add a release note about it (including that they can use the `flow.NewImmediateProgressReporter` as a replacement)?

Reply:
Yep, will do.